src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final String defaultColor = ColorSupplier.Color.WHITE.toString();
+    private final int defaultRadius = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        Shape shape = Shape.values()[random.nextInt(Shape.values().length)];
+        String color = colorSupplier.getRandomColor();
+        int firstSize = random.nextInt(Figure.MAX_SIDE);
+        switch (shape) {
+            case CIRCLE:
+                figure = new Circle(color, firstSize);
+                break;
```

Comment:
```suggestion
                return new Circle(color, firstSize);
```
the same for other case blocks + create for each figure a separate method that will be returning a random figure, for example, getRandomSquare() and so on